camel/societies/role_playing.py

```diff
@@ -365,14 +365,10 @@ def init_chat(self) -> Tuple[BaseMessage, List[BaseMessage]]:
         # Send the system messages again to the agents using chat messages
         assistant_msg = BaseMessage.make_assistant_message(
             role_name=self.assistant_sys_msg.role_name,
-            content=(f"{self.user_sys_msg.content}. "
+            content=(f"{self.assistant_sys_msg.content}. "
                      "Now start to give me instructions one by one. "
                      "Only reply with Instruction and Input."))
-
-        user_msg = BaseMessage.make_user_message(
-            role_name=self.user_sys_msg.role_name,
-            content=f"{self.assistant_sys_msg.content}")
-        assistant_response = self.assistant_agent.step(user_msg)
+        assistant_response = self.assistant_agent.step(assistant_msg)
```

Comment:
The `assistant_msg` is the message to be sent to the user by the assistant agent. So I guess the `self.user_sys_msg` is needed to be sent. Do you confirm this change is correct?